script.js

```diff
@@ -0,0 +1,152 @@
+const mobileMenuDiv = document.querySelector('#mobile-menu');
+mobileMenuDiv.className = 'mobile-menu-hidden';
+let anchorTag = document.createElement('a');
+anchorTag.setAttribute('href', '/');
+anchorTag.style.display = 'flex';
+anchorTag.style.justifyContent = 'flex-end';
+anchorTag.style.marginBottom = '10%';
+anchorTag.style.marginRight = '10%';
+const closingIcon = document.createElement('IMG');
+closingIcon.setAttribute('src', 'imgs/Icon.png');
+closingIcon.setAttribute('alt', 'icon image');
+closingIcon.id = 'close-button';
+anchorTag.appendChild(closingIcon);
+mobileMenuDiv.appendChild(anchorTag);
+anchorTag = document.createElement('a');
+anchorTag.textContent = 'Portfolio';
+anchorTag.setAttribute('href', '#actual-work-section');
+anchorTag.className = 'menu-links';
+mobileMenuDiv.appendChild(anchorTag);
+anchorTag = document.createElement('a');
+anchorTag.textContent = 'About';
+anchorTag.setAttribute('href', '#menu-about');
+anchorTag.className = 'menu-links';
+mobileMenuDiv.appendChild(anchorTag);
+anchorTag = document.createElement('a');
+anchorTag.textContent = 'Contact';
+anchorTag.setAttribute('href', '#menu-contact');
+anchorTag.className = 'menu-links';
+mobileMenuDiv.appendChild(anchorTag);
+const menuButton = document.querySelector('#menu-button');
+function display() {
+  const targetedDiv = document.querySelector('.mobile-menu-hidden');
+  targetedDiv.classList.add('show');
+  menuButton.style.display = 'none';
+}
+
+menuButton.addEventListener('click', display);
+const closeButton = document.querySelector('#close-button');
+function closePop() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

Reply:
This issue is not properly resolved